packages/@secretlint/formatter/src/index.ts

```diff
@@ -7,7 +7,14 @@ import {
 } from "@textlint/linter-formatter";
 import { SecretLintCoreResult } from "@secretlint/types";
 import terminalLink from "terminal-link";
+import { FormatterConfig } from "./types";
+import { moduleInterop } from "@textlint/module-interop";
 
+const fs = require("fs");
```

Comment:
Can you use `import` instead of `require`.

Probably, `@types/try-resolve` is missing. 
We can use `@ts-expect-error` directive.

```j
// @ts-expect-error: no @types
import tryResolve from "try-resolve"
```